Sources/SotoCore/Doc/Region.swift

```diff
@@ -167,3 +167,70 @@ extension AWSPartition {
         }
     }
 }
+
+// allows to create a Region from a String
+// this initialializer returns an Optional.
+// it is non nil for valid region name only
+public extension Region {
```

Comment:
Can you change the comment to indicate this will only create a Region if the provided region name is a valid AWS region name. We already have an initializer that allows you create a Region from a String.

Reply:
done